Source/carthage/main.swift

```diff
@@ -9,8 +9,14 @@
 import CarthageKit
 import Commandant
 import Foundation
-import Result
+import ReactiveCocoa
 import ReactiveTask
+import Result
+
+guard ensureGitVersion().first()?.value == true else {
+	fputs("Carthage requires git \(CarthageRequiredGitVersion) or later. Please update your git version at first.\n", stderr)
```

Comment:
Can we drop the second sentence and just make it `Carthage requires git \(CarthageRequiredGitVersion) or later.`?